src/api/index.ts

```diff
@@ -165,6 +165,11 @@ function createApp() {
         .array(z.nativeEnum(NetworkId))
         .nonempty()
         .or(z.nativeEnum(NetworkId)), // singleton arrays sometimes serialize as single values
+      supportedPools: z
```

Comment:
should we also get app ids from the request? seems like it's not optimal to get all positions for supported apps and then filter down